test/src/k-provider/ovp/provider.spec.js

```diff
@@ -83,6 +83,23 @@ describe('OVPProvider.partnerId:1082342', function () {
     );
   });
 
+  it('should be able to get media config by referenceId', done => {
+    provider = new OVPProvider({partnerId: partnerId, uiConfId: 38621471}, playerVersion);
+    sinon.stub(MultiRequestBuilder.prototype, 'execute').callsFake(function () {
+      this.requests[1].params.filter.referenceIdEqual.should.equal('1_rsrdfext');
+      done();
+      return new Promise(() => {});
+    });
+    provider.getMediaConfig({referenceId: '1_rsrdfext'}).then(mediaConfig => {
+      try {
```

Comment:
this is unreachable code. the test already done in the `execute` stub. move the `try catch` there

Reply:
Please check now